legal-api/src/legal_api/services/filings/validations/change_of_receivers.py

```diff
@@ -31,33 +31,30 @@
 # CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
-"""Validation for the Cease Receiver filing."""
-from http import HTTPStatus
+"""Validation for the Change of Receivers filing."""
+from datetime import datetime, timezone
 from typing import Optional
 
-from flask_babel import _ as babel
-
 from legal_api.errors import Error
 from legal_api.models import Business, PartyRole
-from legal_api.utils.datetime import datetime, timezone
-
+from legal_api.services.filings.validations.common_validations import validate_parties_addresses
 
-def validate(business: Business, cease_receiver: dict) -> Optional[Error]:
-    """Validate the Cease Receiver filing."""
-    if not business or not cease_receiver:
-        return Error(HTTPStatus.BAD_REQUEST, [{"error": babel("A valid business and filing are required.")}])
-    msg = []
-    msg.extend(validate_party(business, cease_receiver))
 
-    if msg:
-        return Error(HTTPStatus.BAD_REQUEST, msg)
-    return None
+def validate(business: Business, filing_json: dict) -> Optional[Error]:
```

Comment:
I believe we need to do this validation for relationship as well https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/services/filings/validations/common_validations.py#L272